qmetaobject/src/qtdeclarative.rs

```diff
@@ -428,7 +428,12 @@ pub fn qml_register_type<T: QObject + Default + Sized>(
             meta_object
         );
 
-        int parserStatusCast = meta_object && meta_object->inherits(&QQuickItem::staticMetaObject)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 7, 0)
+        bool inherits = meta_object->inherits(&QQuickItem::staticMetaObject);
+#else
+        bool inherits = true;
```

Comment:
In Qt 5.6 we could make a loop and compere QMetaObject parent.
Probably you can write an helper function that does that.
But well, since this is just for a safety assert, i guess it is ok to skip the test.

Reply:
Hmm, this idea just came from my forks patch that just dropped the checks. Hehe. I was lazy at the time.

Do you have any pointers to the Qt docs how I could do that?

Reply:
Ah, I [found it](https://doc.qt.io/qt-5/qmetaobject.html#superClass), I think. I'll get on that.

Reply:
Push that in cdc67df. I've tested it by swapping `<=` and `>=` and ran `cargo test` on a Qt 5.15 installation, haven't tested yet on my phone, but I don't expect anything weird.

Reply:
You can probably pass all inherits checks through that function and do only one version check inside it. :)

Reply:
Was thinking that too, and I dismissed it because of a reasoning that I didn't make well. I'll change it!

Reply:
8590a14e5ec8f088f48274d59cb18b93f91775c7